pkg/loader/oauth_loader.go

```diff
@@ -0,0 +1,148 @@
+package loader
+
+import (
+	"encoding/json"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/hellofresh/janus/pkg/oauth"
+	"github.com/hellofresh/janus/pkg/proxy"
+	"github.com/hellofresh/janus/pkg/store"
+	"github.com/rs/cors"
+)
+
+// OAuthLoader handles the loading of the api specs
+type OAuthLoader struct {
+	register *proxy.Register
+	storage  store.Store
+	subs     *store.Subscription
+}
+
+// NewOAuthLoader creates a new instance of the Loader
+func NewOAuthLoader(register *proxy.Register, storage store.Store, subs *store.Subscription) *OAuthLoader {
+	return &OAuthLoader{register, storage, subs}
+}
+
+// LoadDefinitions loads all oauth servers from a datasource
+func (m *OAuthLoader) LoadDefinitions(repo oauth.Repository) {
+	oAuthServers := m.getOAuthServers(repo)
+	m.RegisterOAuthServers(oAuthServers, repo)
+}
+
+// RegisterOAuthServers register many oauth servers
+func (m *OAuthLoader) RegisterOAuthServers(oauthServers []*oauth.Spec, repo oauth.Repository) {
+	log.Debug("Loading OAuth servers configurations")
+
+	for _, oauthServer := range oauthServers {
+		if m.subs != nil {
```

Comment:
Add some log here, like `log.WithField("name", oauthServer.Name).Debug("Registering OAuth server")` and same to the end of the registering the server process.